XenModel/Messages.resx

```diff
@@ -11991,6 +12012,27 @@ To learn more about the [XenServer] Dynamic Workload Balancing feature or to sta
   <data name="URI_REGEX" xml:space="preserve">
     <value>^(http|https|file|ftp)://*</value>
   </data>
+  <data name="USB" xml:space="preserve">
+    <value>USB</value>
+  </data>
+  <data name="USBLIST_DISABLE_PASSTHROUGH" xml:space="preserve">
+    <value>Disable passthrough</value>

```

Comment:
As this is the text apearing on a button, a hotkey is needed. 
Same for USBLIST_ENABLE_PASSTHROUGH

Reply:
Will fix.

Reply:
Considering the string is the same as dialog title, so instead of changing it in Messages.resx, I changed in assignation:
>buttonUsage.Text = String.Format("&{0}", Messages.USBLIST_DISABLE_PASSTHROUGH);

Reply:
We can't have the hotkey hardcoded to the first letter, because it might need to be different in other languages, so it need to be in the resources. If this means two strings, then so be it.